src/brsTypes/components/RoSGNode.ts

```diff
@@ -69,23 +71,18 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
     private fields = new Map<string, Field>();
     private children: RoSGNode[] = [];
     private parent: RoSGNode | BrsInvalid = BrsInvalid.Instance;
-    readonly builtInFields = [
+    readonly builtInFields: FieldModel[] = [
         { name: "change", type: "roAssociativeArray" },
         { name: "focusable", type: "boolean" },
         { name: "focusedChild", type: "node" },
         { name: "id", type: "string" },
     ];
 
     constructor(members: AAMember[], readonly type: string = "Node") {
```

Comment:
So, do we want this class to have the name `"Node"`, or `"roSGNode"`?

Reply:
I think _Node_ is fine for now.  It's simple to swap if we change our minds.